concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
+1	Bowern-2012-348-1	1dl.excl	2636	WE TWO (EXCLUSIVE)	
+2	Bowern-2012-348-2	1dl.incl	2637	WE TWO (INCLUSIVE)	
+3	Bowern-2012-348-3	3dl	2635	THEY TWO	
+4	Bowern-2012-348-4	Bloodwood	3007	EUCALYPT	
+5	Bowern-2012-348-5	Blue-Tongue Lizard	632	LIZARD	mapping to LIZARD as it's the only Lacertilia in the list
```

Comment:
This kind of mapping may be allowed in rare cases, in this one, I'd say it is okay.

Reply:
Good, I almost opened an issue to ask about it. My impression was that, in general, it is better to have Concepticon as stable as possible, favoring corrections on concept lists, am I correct?